utils/phoneme/phonemeAnalyzer.ts

```
@@ -0,0 +1,127 @@
+    analyzer: Meyda.MeydaAnalyzer;
+    microphone: MediaStreamAudioSourceNode;
+  }>;
+  startAnalysis: () => void;
+  stopAnalysis: () => void;
+}
+
+/**
+ * Initialize Meyda analyzer for live audio feature extraction.
+ */
+export function setupAudioAnalysis(audioContext?: AudioContext): AudioAnalyzer {
+  const context = audioContext ?? new (window.AudioContext || (window as any).webkitAudioContext)();
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)